infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -575,6 +617,39 @@ private void handleAllocation(CorfuPayloadMsg<TokenRequest> msg, ChannelHandlerC
                 new TokenResponse(token, backPointerMap.build())));
     }
 
+    /**
+     * This method handles the request of streams address maps.
+     * It computes the addresses requested for each stream
+     * and returns this map under a StreamsAddressResponse message.
+     */
+    @ServerHandler(type = CorfuMsgType.STREAMS_ADDRESS_REQUEST)
+    private synchronized void handleStreamsAddressRequest(CorfuPayloadMsg<StreamsAddressRequest> msg,
```

Comment:
You need not have synchronized on this as the Sequencer is single threaded.
Also, can you change this to public just like other handlers.

Reply:
yes, you're right, I missed this one. Thanks